src/anyio/_backends/_asyncio.py

```diff
@@ -877,14 +878,16 @@ def stderr(self) -> Optional[abc.ByteReceiveStream]:
         return self._stderr
 
 
-async def open_process(command, *, shell: bool, stdin: int, stdout: int, stderr: int):
+async def open_process(command, *, shell: bool, stdin: int, stdout: int, stderr: int,
+                       cwd: Union[str, Path, None] = None,
```

Comment:
isn't this 
```suggestion
                       cwd: Union[str, bytes, os.PathLike, None] = None,
```

Reply:
Argh Python makes me loopy sometimes.  Yes you're right per https://docs.python.org/3/glossary.html#term-path-like-object